test/Main.purs

```diff
@@ -68,6 +68,10 @@ main = launchAff_ $ runSpec [consoleReporter] do
       let opts = { method: M.putMethod }
       result <- attempt $ fetch (M.URL "https://www.google.com") opts
       isRight result `shouldEqual` true
+    it "patch works" do
+      let opts = { method: M.patchMethod }
+      result <- attempt $ fetch (M.URL "https://www.google.com") opts
+      isRight result `shouldEqual` true
```

Comment:
ℹ️ Added this test for consistency with other method tests, but executing unsupported methods on google's services will return `405` - independent of the method being a valid http method or not.

A service like <https://postman-echo.com/put> could be handy, but would be a third party dependency.